erts/configure.in

```diff
@@ -2771,6 +2771,21 @@ if test "$cross_compiling" != "yes" && test X${enable_hipe} != Xno; then
   fi
 fi
 
+dnl Check to disable -fPIE and friends for HiPE on amd64
+if test X${enable_hipe} = Xyes && test X$ARCH = Xamd64; then
+   AC_TRY_COMPILE(, [#ifdef defined(__pie__) || defined(__PIE__)
```

Comment:
Thanks.
Believe it or not but I actually tested *both* branches of AC_TRY_COMPILE.
#ifdef define(..) succeeded with just a warning "extra tokens at end of #ifdef directive" (that I never saw).
Then I changed it to !defined(..) and that did not compile as expected.